dev-docs/source/howto/schemas.rst

```diff
@@ -0,0 +1,30 @@
+Schema Development
```

Comment:
See: https://github.com/Amsterdam/dso-api/pull/261/files#r629214216 and ditch the `code-block:: bash` directives